controllers/controllers.go

```diff
@@ -153,6 +153,10 @@ func (r *instanceReconciler) reconcileKubeletClientCA(ctx context.Context, bundl
 	// matches the signer subject.
 	kubeAPIServerServingCABytes, err := certificates.MergeCAsConfigMaps(initialCAConfigMap, bundleCAConfigMap,
```

Comment:
Consider adding a unit test for `reconcileKubeletClientCA()` to check the propagation of a validation error. i.e., passing an empty subject.

Reply:
> Introducing an early return of the error is helpful, but I don't feel it was causing any issues, given there is a [fail-check](https://github.com/alinaryan/windows-machine-config-operator/blob/master/pkg/nodeconfig/nodeconfig.go#L600-L603) for empty contents.

If a function returns an error it should always be checked.

Reply:
Thinking of adding a unit test to check that the correct subject signer is in both bundles if we determine that is the cause of the bug: https://issues.redhat.com/browse/OCPBUGS-22237

I could also add what you're suggesting in that same PR.